lib/libevent/libevent.nit

```diff
@@ -484,6 +535,28 @@ class ConnectionFactory
 		bev.setcb conn
 	end
 
+	# Accept a connection on a Unix socket
+	fun accept_unix_connection(listener: ConnectionListener, fd: Int, addrin: Pointer, socklen: Int)
+	do
+		var base = listener.base
+		var bev = new NativeBufferEvent.socket(base, fd, bev_opt_close_on_free)
+		var addr=""
```

Comment:
It seems this is the only difference between the regular accept_connection and the accept_unix_connection.
Maybe these two functions can be merged? Is there a way to check at runtime which type of Connection we were listening to?

Reply:
Best way to merge these methods would be to add two indirections. This is a sensitive point since they're called with a C callback. I'll investigate this option.